source/_integrations/husqvarna_automower.markdown

```diff
@@ -161,6 +161,11 @@ The integration will create the following sensors:
 - Total searching time
 - Work area (if available). For example: *My lawn*, *Front lawn*, *Back lawn*
 
+For each work area with activated systematic mowing these sensors are created:
+
+- Progress
```

Comment:
Progress in percent?
Last time completed: is that something like "2 days ago"?

Reply:
> Progress in percent? 

Yes

> Last time completed: is that something like "2 days ago"?

Yes, it's a datetime object. So the frontend renders something like that.

Reply:
:pray: thank you